drivers/wifi/nrf700x/Kconfig.nrf700x

```diff
@@ -43,11 +40,11 @@ config NET_CONFIG_INIT_TIMEOUT
 	default 0 if NET_CONFIG_SETTINGS
 
 config NRF700X_ON_QSPI
-	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF7002_QSPI))
+	def_bool NRF7000_ON_QSPI
```

Comment:
Though ultimately its what we want, this commit still breaks the build, it should be `NRF7002_ON_QSPI || NRF7000_ON_QSPI`, NRF7002 should work with all commits.

Reply:
Done.